packages/web/src/components/NotFound.tsx

```diff
@@ -8,31 +9,36 @@ import SearchForm from "./SearchForm";
 import Typography from "@material-ui/core/Typography";
 import { colors } from "@upswyng/common";
 
-const NotFound = () => (
-  <Container>
-    <PageBanner color={colors.orangeDark}>
-      <Typography variant="h1">Page not Found</Typography>
-    </PageBanner>
-    <Grid container direction="column" spacing={4}>
-      <Grid item>
-        <Typography component="p" variant="h3">
-          Whoops! We&apos;re having trouble finding that page.
-        </Typography>
+const NotFound = () => {
+  const { t } = useTranslation(["notFound"]);
+  return (
+    <Container>
+      <PageBanner color={colors.orangeDark}>
+        <Typography variant="h1">{t("pageTitle")}</Typography>
+      </PageBanner>
+      <Grid container direction="column" spacing={4}>
+        <Grid item>
+          <Typography component="p" variant="h3">
+            {t("message")}
+          </Typography>
+        </Grid>
+        <Grid item>
+          <SearchForm />
+        </Grid>
+        <Grid item>
+          <Typography>
+            <Trans
+              ns="notFound"
+              i18nKey="goHome"
+              components={{
+                MuiLink: <MuiLink component={RouterLink} to="/" />,
+              }}
```

Comment:
Huh, neat I hadn't used the `<Trans/>` component before 👍

Reply:
I am seeing that it doesn't re-render on language change though. What are your thoughts on this vs breaking this into separate keys instead with the `t` function instead?

Reply:
That caught my eye as well and struck me as very odd, but from a bit more googling ([eg](https://github.com/i18next/react-i18next/issues/458)), it seems like that only applies if you're using i18next without HOCs or... something?
It shouldn't apply for us, and just to double check I tried toggling and it seems to rerender correctly!
![i18nReRender](https://user-images.githubusercontent.com/20876393/109854265-147f2880-7c14-11eb-9486-74021e1d32bc.gif)